Policies/github-terms-of-service.md

```diff
@@ -177,7 +191,7 @@ If you’d like to use GitHub’s trademarks, you must follow all of our tradema
 #### 3. License to GitHub Policies
 This Agreement is licensed under the [Creative Commons Attribution license](https://creativecommons.org/licenses/by/4.0/). You may use it freely under the terms of the Creative Commons license.
 
-### G. API Terms
+### H. API Terms
 **Short version:** *You agree to these Terms of Service, plus this Section G, when using any of GitHub's APIs (Application Provider Interface), including use of the API through a third party product that accesses GitHub.*
```

Comment:
There is really no need for the `plus this Section G,` clause, and it would also need updating to reflect it is now Section H.